app/views/layouts/application.html.erb

```diff
@@ -1,16 +1,116 @@
 <!DOCTYPE html>
```

Comment:
i have sorted this out and started using Slim, took a while but will get the hang of it for sure, much more neater